lib/src/common/model/repos_commits.dart

```diff
@@ -7,119 +7,153 @@ part of github.common;
 /// in [RepositoryCommit] "github details", in [GitCommit] "git details".
 class RepositoryCommit {
   /// API url.
-  String url;
+  final String url;
 
   /// Commit SHA
-  String sha;
+  final String sha;
 
   /// Url to Commit Page
   @JsonKey(name: "html_url")
-  String htmlUrl;
+  final String htmlUrl;
 
   /// Comments url.
   @JsonKey(name: "comments_url")
-  String commentsUrl;
+  final String commentsUrl;
 
   /// A reference to the raw [GitCommit].
-  GitCommit commit;
+  final GitCommit commit;
 
   /// Commit Author
-  User author;
+  final User author;
 
   /// Commit Committer.
-  User committer;
+  final User committer;
```

Comment:
These changes to final are technically breaking changes since if someone has a reference to this class and has code that tries to set one of these fields from outside the class, they can no longer do it.
Since these changes are probably the result of the linter recommending to use final and they aren't strictly necessary, I would prefer to leave them without final.

Reply:
Right. The reason why I made these changes is that classes in models folder are to create in Dart json Objects retrieved from the GitHub API. So the user shouldn't have to edit them manually. Plus, it could introduce some bugs to do so if the user would pass it as arguments in methods to make other API calls.